lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_about/pokemon_about_page.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_moves/pokemon_moves_page.dart';
+import 'package:pokedex/features/pokemon_details/pokemon_stats/pokemon_stats_page.dart';
+import 'package:pokedex/features/pokemon_overview/pokemon_overview_connector.dart';
+import 'package:pokedex/utils/strings.dart' as str;
+
+class PokemonDetailsPage extends StatelessWidget {
+  const PokemonDetailsPage({
+    Key? key,
+  }) : super(key: key);
+
+  static const String route = '/details';
+  static const String routeName = 'details';
+
+  @override
+  Widget build(BuildContext context) {
+    final appBarViews = <Widget>[
+      const PokemonAboutPage(),
+      const PokemonStatsPage(),
+      const PokemonMovesPage(),
+    ];
+    final appBarItems = <Tab>[
+      const Tab(child: Text(str.aboutTitle)),
+      const Tab(child: Text(str.statsTitle)),
+      const Tab(child: Text(str.movesTitle)),
+    ];
+    return DefaultTabController(
+      length: appBarItems.length,
+      child: Scaffold(
+        appBar: AppBar(
+          leading: IconButton(
+            icon: const Icon(Icons.arrow_back),
+            onPressed: () {
+              context.go(PokemonOverviewConnector.route);
+            },
+          ),
+          title: const Text(
+            str.pokemonDetailsTitle,
+          ),
+        ),
+        body: Column(
+          children: <Widget>[
+            const SizedBox(
+              height: 320.0,
+              child: Center(
+                /// TODO: place image, name, type, of specific pokemon here
+                child: Text(
+                  'Place Pokemon Basic Details Here',
```

Comment:
always don't leave magic strings/ints/values hanging independently even though it's just a temporary - it's recommended to have it in a private constant then 🙂